sub/dec_sub.c

```diff
@@ -149,7 +149,7 @@ static struct sd *init_decoder(struct dec_sub *sub)
             .preload_ok = true,
         };
 
-        if (sd->driver->init(sd) >= 0)
+        if (sd->driver->init(sd, sub->order) >= 0)
```

Comment:
Instead of passing `sub->order` everywhere, it's probably easier to just add `order` to the `sd` struct at this point.

Reply:
Done.